test/src/templates/test_utils.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter_deck/flutter_deck.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+class SlideTester {
+
+  SlideTester({
```

Comment:
Probably you can use const there:
```suggestion
  const SlideTester({
```